designsafe/apps/rapid/openTopo_Data/scripts/centerpoint_Requirements.txt

```
@@ -0,0 +1,3 @@
+requests==2.31.0
+geojson==3.0.1
+shapely==2.0.1
```

For my version of python, I needed to tweak this a bit.
```suggestion
requests==2.31.0
geojson>=3.0.1,<4.0.0
shapely>=2.0.1,<3
```

---

This is a suggestion comment, so if you agree with it you can click commit suggestion and Github will commit the change with you/me as authors.  For small suggestions (especially spelling errors etc), its really useful.